app/views/posts/index.html.erb

```diff
@@ -7,4 +7,7 @@
 
 <p style="display: flex; justify-content: center; text-shadow: 2px 2px 4px grey;">
   <span style="border: 3px solid grey; padding: 4px"> <%= link_to 'New Post', new_user_post_path(@user) %></span>
+  <% if can? :read, @post %>
+  <% link_to "View", @post%>
+  <% end %>
 </p>
```

Comment:
- [ ] I also cannot access this page, as an admin. Kindly fix that.

  ![image](https://github.com/bhushan354/Blog-App-Rails/assets/36542986/cfa2964c-b6cc-4766-8311-6cbc2ab4b231)